pyproject.toml

```diff
@@ -30,6 +30,8 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
 space_packet_parser = ">=4.0.2,<5"
+pandas = ">=2.0.0,<3"
+openpyxl = ">=3.0.7,<4"
```

Comment:
These are optional dependencies since the xtce stuff is going to live in extra tools and isn't required every time we run the code. Maybe a new section that could be installed called `xtce` or `tools`?